haskell/src/Lorentz/Contracts/Stablecoin.hs

```diff
@@ -450,26 +432,16 @@ registryContract =
         |]
   )
 
--- | Parse the contract-metadata contract.
-contractMetadataContract :: T.Contract (ToT ()) (ToT ContractMetadataRegistryStorage)
-contractMetadataContract =
-  $(case readContract @(ToT ()) @(ToT ContractMetadataRegistryStorage) contractMetadataRegistryContractPath $(embedStringFile contractMetadataRegistryContractPath) of
-      Left e -> fail (pretty e)
-      Right _ ->
-        [|
-          -- Note: it's ok to use `error` here, because we just proved that the contract
-          -- can be parsed+typechecked.
-          either (error . pretty) snd $
-            readContract
-              contractMetadataRegistryContractPath
-              $(embedStringFile contractMetadataRegistryContractPath)
-        |]
-  )
-
 ----------------------------------------------------------------------------
 -- TZIP-16 Metadata
 ----------------------------------------------------------------------------
 
+jfield :: MText
+jfield = [mt|metadataJSON|]
+
+tezosStorageScheme :: Text
+tezosStorageScheme = "tezos-storage"
```

Comment:
This is a very small request, but since you have an MR in `morley-metadata` open I would use it to also move this there.

I doubt it would make much of a difference, but indeed this is also a shared value AFAIU and it's already defined in 
`Lorentz.Contracts.Spec.TZIP16Interface.URI`, just not exported.

Reply:
Moved. I have also made some changes to use the infra from morley-metadata to make the storage uri.